### ISSUE TITLE
Create an expansion item estimation id list

### ISSUE THREAD

**Issue by xackery** [xackery/peq-expansions#9]
I wouldn't invest a ton of time into this, but it can help us baseline what items on what era. I would just have a csv file with like:

itemid,expansion

**Comment by nathanhicks** [comment]
@xackery Do you just want me to push this into the `peq/` directory when it's finished?

**Comment by xackery** [comment]
Yeah, that's fine. peq is grouping anything that will impact peq in future.. anything out of it is more like adjustments for an expansion-specific edit that won't ever make it to peq.

**Comment by nathanhicks** [comment]
Update - item csv is sitting in the peq/ folder with all the item ID's from peq_beta database.  Getting started soon on trying to tag them with the expansion.
